cli/cli/Commands/Services/ServicesListCommand.cs

```diff
@@ -55,220 +46,131 @@ public override async Task Handle(ServicesListCommandArgs args)
-				foreach (var sd in serviceDefinitions)
-				{
-					var beamoIdMarkup = new Markup($"[green]{sd.BeamoId}[/]");
-					var imageIdMarkup = new Markup($"{sd.TruncImageId}");
-					var shouldBeEnabledOnDeployMarkup = new Markup(sd.ShouldBeEnabledOnRemote ? "[green]Enable[/]" : "[red]Disable[/]");
-					var isRemoteOnlyMarkup = new Markup(_localBeamo.VerifyCanBeBuiltLocally(sd) ? "[green]True[/]" : "[red]False[/]");
-
-					IRenderable containersRenderable;
-					var existingServiceInstances = runningServiceInstances.Where(si => si.BeamoId == sd.BeamoId).ToList();
-					var hasNoRunningInstances = existingServiceInstances.Count == 0;
-					if (hasNoRunningInstances)
-					{
-						containersRenderable = new Markup("[yellow]-------------------[/]");
-					}
-					else
-					{
-						var containersTable = new Table();
-						var containerNameColumn = new TableColumn("Container Name");
-						var containerStatusColumn = new TableColumn("Container Status");
-						var containerPortMappingsColumn = new TableColumn("Container Port Mappings");
-						containersTable.AddColumn(containerNameColumn);
-						containersTable.AddColumn(containerStatusColumn);
-						containersTable.AddColumn(containerPortMappingsColumn);
-						foreach (var existingServiceInstance in existingServiceInstances)
-						{
-							var containerNameMarkup = new Markup(existingServiceInstance.ContainerName);
-							var containerStatusMarkup = new Markup(existingServiceInstance.IsRunning ? "[green]On[/]" : "[red]Off[/]");
-
-							var portMappings = existingServiceInstance.ActivePortBindings.Select(p => $"{p.LocalPort}:{p.InContainerPort}");
-							var containerPortMappingMarkup = new Markup(string.Join(", ", portMappings));
-							containersTable.AddRow(containerNameMarkup, containerStatusMarkup, containerPortMappingMarkup);
-						}
-
-						containersRenderable = containersTable;
-					}
-
-					localServiceListResult.AddService(sd.BeamoId,
-						sd.ShouldBeEnabledOnRemote,
-						!hasNoRunningInstances,
-						sd.Protocol.ToString(),
-						sd.ImageId,
-						"",
-						"",
-						new[] { "" },
-						new[] { "" },
-						dependenciesDict[sd].Select(dep => dep.name),
-						sd.ProjectDirectory);
-					table.AddRow(new TableRow(new[] { beamoIdMarkup, imageIdMarkup, containersRenderable, shouldBeEnabledOnDeployMarkup, isRemoteOnlyMarkup, }));
-				}
-
-				this.SendResults(localServiceListResult);
-				if (serviceDefinitions.Count > 0)
-				{
-					AnsiConsole.Write(table);
-				}
-				else
-				{
-					AnsiConsole.WriteLine("No services found");
-				}
+				AnsiConsole.Write(table);
 			}
 			else
 			{
-				AnsiConsole.WriteLine(JsonConvert.SerializeObject(_localBeamo.BeamoManifest, Formatting.Indented));
+				AnsiConsole.WriteLine("No services found");
 			}
+		}
+		else
+		{
+			AnsiConsole.MarkupLine($"[green]Current Manifest[/]");
+			AnsiConsole.WriteLine(JsonConvert.SerializeObject(_localBeamo.BeamoManifest, Formatting.Indented));

```

Comment:
Talk to Allister and Chris, because with this change, it won't be possible to get the remote manifest json through the `services ps` anymore, how we fix this case?

That can be done with this:
```
var manifest = await _beamo.GetManifest();
AnsiConsole.WriteLine(JsonConvert.SerializeObject(manifest Formatting.Indented));
```